frontend/test/app/actions/newsletter-form-test.js

```diff
@@ -0,0 +1,83 @@
+import fetchMock from 'fetch-mock';
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { basketUrl } from '../../../src/app/actions/newsletter-form';
+
+const FAILED = 'newsletterFormSetFailed';
+const SUBMITTING = 'newsletterFormSetSubmitting';
+const SUCCEEDED = 'newsletterFormSetSucceeded';
+
+const MOCK_EMAIL = 'foo@bar.com';
+
+const setUp = responseCode => {
+  fetchMock.post(basketUrl, responseCode);
+  return {
+    dispatch: sinon.spy(),
+    subscribe: require('../../../src/app/actions/newsletter-form').default.newsletterFormSubscribe
```

Comment:
Has to be imported after the `fetchMock.post` call.

A little ugly, but everything else mocks _so cleanly_ that it's probably fine.